Automattic/sensei
# Fetch sensei extensions from sensei site

Fixes #3044

### Changes proposed in this Pull Request

* Fetch the extensions from `https://senseilms.com/wp-json/senseilms-products/1.0/search?category=setup-wizard-extensions` - I created this new category `setup-wizard-extensions` to add only the extensions that we want to appear as suggestion.

### Testing instructions

* Go to `/wp-admin/admin.php?page=sensei_onboarding&step=features` and check if the extensions are being listed.

### Some questions

* Should we really override the titles? Or could we rename it in the `senseilms.com`?
* Do we need to worry about the [dynamic data](https://senseilms.com/wp-json/senseilms-products/1.0/search?category=setup-wizard-extensions) translations and currency conversion?
* Are good the links coming from the JSON? They are pointing to senseilms.com and redirecting to WC.com or WP.org.
* About the sorting, do we have a specific rule to apply in the array? I had thought about creating an array with the sort, but if we add more extensions in the future it would not appear in the order that we want.
